homeassistant/components/ring/__init__.py

```diff
@@ -32,6 +45,9 @@ def setup(hass, config):
     conf = config[DOMAIN]
     username = conf[CONF_USERNAME]
     password = conf[CONF_PASSWORD]
+    scan_interval = conf[CONF_SCAN_INTERVAL]
+
+    ring_devices = []
```

Comment:
We can remove this.

Reply:
thought it needed declaring. Python is fun.